practices/communication.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Communication
+description: Best practices to have a clear communication
+---
+
+# Communication
+
+## Async Communication
+
+At Loadsmart, the engineering team works 100% remotely. We believe async communication is key for our day-to-day work.
+
+### Business Time Visibility
+
+People are working from different time zones, and we need to be aware of and respect it. For example, if we don't know your timezone or your business time, we don't know when you would be able to attend a meeting or answer a message. Two tips here:
+You can set your business time using Google Calendar. With that, everybody will see it when scheduling a meeting with you.
+Add this information to your Slack profile. It will be available with a single click on your Slack picture. We recommend this format `<job title> @ <squad/team/department> - Working hours: X AM - Y PM (<timezone>)`
+
+### No Hello
+
+When sending a message, like in Slack, we usually try to be polite by not jumping right into the request. This is not a good practice and costs time. The [No Hello](https://nohello.net) website presents a better way to do it. Please take a look.
+<div align="center">
+  <img alt="No Hello" src="./no-hello.png" width="600" />
+  <br/>
+  <i>Image by <a href="https://nohello.net">No Hello</a></i>
+</div>
+
+### Slack Notifications
+
+It is common and useful to have Slack installed on our phones. With this, we got the notifications as well. They are very useful when we are working, but they can be very unhealthy. Receiving notifications when you are out of your business time will prevent you from disconnecting and resting. Within some time, it will cause a lot of stress. That's why we recommend setting Slack to send mobile notifications during your business hours or for really urgent channels only.
```

Comment:
```suggestion
It is common to have Slack installed on our phones. But that comes with notifications too. Notifications are useful while you are working, but they are certainly unhealthy when you get bombarded by them outside of working hours. It prevents you from disconnecting and resting. Within time, it may increase your level of stress. That's why we recommend configuring Slack to send mobile notifications during your business hours only or for messages sent in urgent channels.
```